src/Api.hs

```diff
@@ -3,13 +3,19 @@
 
 module Api (app) where
 
-import           Control.Monad.Reader (runReaderT)
-import           Servant              ((:<|>) ((:<|>)), Proxy (Proxy), Raw,
-                                       Server, serve, serveDirectoryFileServer)
-import           Servant.Server
-
-import           Api.User             (UserAPI, userApi, userServer)
-import           Config               (AppT (..), Config (..))
+import Control.Monad.Reader (runReaderT)
+import Servant
+       ( (:<|>)((:<|>))
+       , Proxy(Proxy)
+       , Raw
+       , Server
+       , serve
+       , serveDirectoryFileServer
+       )
+import Servant.Server
+
+import Api.User (UserAPI, userApi, userServer)
+import Config (AppT(..), Config(..))
```

Comment:
We no longer align the explicit import lists along the longest module name. This is less noisy, because adding a new module import that is longer than any others will no longer trigger a reformat across all the imports.